sentry.gradle

```diff
@@ -169,7 +202,21 @@ gradle.projectsEvaluated {
             group = 'sentry.io'
 
             delete sourcemapOutput
-            delete "$buildDir/intermediates/assets/release/index.android.bundle.map" // react native default bundle dir
+            delete "$buildDir/intermediates/assets/release/index.android.bundle.map"
+            // react native default bundle dir
+        }
+
+        def modulesCleanUpTask = tasks.create(name: nameModulesCleanup, type: Delete) {
+            description = "clean up collected modules generated file"
+            group = 'sentry.io'
+
+            delete modulesOutput
+        }
+
+        def packageTasks = tasks.findAll { task -> task.name.startsWith("package") && !task.name.contains("Debug") && task.enabled }
```

Comment:
Doing the search this way has side effects in some Gradle versions.
See how the Gradle plugin does https://github.com/getsentry/sentry-android-gradle-plugin/blob/fd53b091525447713d93d72f5d6f018c67aedcc8/plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt#L214-L216

Reply:
Also need to check if the `TaskProvider` for this task is available on older versions of the AGP

Reply:
This is not a blocker, but I guess this will trigger some warnings on newer AGP/Gradle versions

Reply:
The Gradle plugin uses `project.findTask`

We are already using this loop in the rn `sentry.gradle` plugin. For the other tasks.